service/compile.sh

```diff
@@ -0,0 +1,40 @@
+if [ -z "${DDC_PATH}" ]; then
+  echo "Please set DDC_PATH so that the Dev Compiler may be found."
+  exit 1
+fi
+
+if [ -z "${MARKDOWN_SERVICE_BUCKET}" ]; then
```

Comment:
Move this check to the bottom – and maybe just print out a message.

It can be useful to just have the code to run, without publishing